tests/tests.py

```diff
@@ -1047,6 +1047,26 @@ def test_apply_abtests(self):
         self.assertEqual(msgs2A, exp2A)
 
 
+class TestRapidProABTestCreatorRegexReplace(unittest.TestCase):
```

Comment:
Very happy that there is a test here. Would you also create a direct test of the 'count' and 'replace' functions?

Reply:
Done.